src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -213,6 +222,58 @@ public void shouldReorderQueueSuccessfully(int firstPosition,
     verifyQueueUpdated(reorderQueue, response);
   }
 
+  @Test
+  public void logRecordEventIsPublished() {
```

Comment:
Maybe we can just add event published assertions to an existing test (for example `shouldReorderQueueSuccessfully`)?

Reply:
Are you fine to leave it as is because all the positive tests are parametrize? As a result different number of log record events are generated. This test covers only one move action and verify content of this particular event.

Reply:
Sure, let's leave it as is